include/fmt/std.h

```diff
@@ -389,6 +390,16 @@ struct formatter<
 #endif
   }
 };
+
+FMT_EXPORT
+template <typename Char>
+struct formatter<std::vector<bool>::reference, Char> : formatter<bool, Char> {
```

Comment:
Maybe need to handle non default allocator

Reply:
We can't deduce Allocator type in this context.

```
/Users/phprus/Devel/fmt/include/fmt/std.h:395:8: error: class template partial specialization contains a template parameter that cannot be deduced; this partial specialization will never be used [-Wunusable-partial-specialization]
struct formatter<typename std::vector<bool, Allocator>::reference, Char>
       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/phprus/Devel/fmt/include/fmt/std.h:394:20: note: non-deducible template parameter 'Allocator'
template <typename Allocator, typename Char>
                   ^
```